### PR TITLE
Changes for compatibility with ts_xml 3.10.2

### PR DESCRIPTION
**Pull request by menanteau**

### CODE REVIEW

File: etc/conf/atTelemetry.yaml [hunk 1]
```diff
@@ -58,21 +58,21 @@ telemetry:
...
-  value: timeStamp
+  value: timeStampAcquisitionStart
   collect_after_event: start_collection_event
 
  DATE-END:
   device: ATCamera
   Stype: Event
   topic: endReadout
-  value: timeStamp
+  value: timeStampAcquisitionStart
```
Comment (patrickingraham) [inline etc/conf/atTelemetry.yaml:75]: shouldn't this be at the end of the acquisition rather than the start?

File: etc/ATSCam/primary_hdu.header [hunk 2]
```diff
@@ -48,7 +48,7 @@ AZEND   =                      / [deg] Telescope azimuth angle at end
 AMEND   =                      / Airmass at end
 
         ---- Image-identifying used to build OBS-ID ----
-TELCODE = 'AT'                 / The code for the telecope
+TELCODE =                      / The code for the telecope (AT/CC/MC)
 CONTRLLR=                      / The controller (e.g. O for OCS, C for CCS)
```
Comment (patrickingraham) [inline etc/ATSCam/primary_hdu.header]: ACCS/CCS

The ACCS is used to control the ATCAM.

File: etc/conf/atTelemetry.yaml [hunk 3]
```diff
@@ -10,7 +10,7 @@ logfile: 'www/HeaderService.log'
 vendor: ITL
 
 # The ts_xml version we are using -- for now this is just curated and need to be changed manually when we change version
-ts_xml: "3.10.0"
+ts_xml: "3.10.2"
 
 # port_number for webserver
 port_number: 8000
```
Comment (patrickingraham) [inline etc/conf/atTelemetry.yaml:16]: This probably shouldn't be hardcoded

File: bin/telemetry_sim_ATS_1VisitNight [hunk 4]
```diff
@@ -180,8 +180,11 @@ if __name__ == "__main__":
         LOGGER.info("Sending starIntegration")
         kw = {'imageSequenceName': args.imageSequenceName,
               'imageName': imageName,
-              'imageIndex': k+1,
-              'timeStamp': timeStamp,
+              'imageNumber': k+1,
```
Comment (patrickingraham) [inline bin/telemetry_sim_ATS_1VisitNight:183]: Sorry, I'm just not quite understanding this.
what is k? and why it is k+1 ?

**Review by patrickingraham — CHANGES_REQUESTED** [review]
Might not require changes but please answer my questions!

File: bin/telemetry_sim_ATS_1VisitNight (hunk 4 above)
Comment (menanteau) [reply bin/telemetry_sim_ATS_1VisitNight:183]: When requesting more than one exposure per sequence.

File: etc/conf/atTelemetry.yaml (hunk 3 above)
Comment (menanteau) [reply etc/conf/atTelemetry.yaml:16]: Yes, but we don't have any other way to get it at the moment.

File: etc/ATSCam/primary_hdu.header (hunk 2 above)
Comment (menanteau) [reply etc/ATSCam/primary_hdu.header]: Is this comment for the CONTRLLR key on the template?

File: etc/conf/atTelemetry.yaml (hunk 1 above)
Comment (menanteau) [reply etc/conf/atTelemetry.yaml:75]: Yes, we discussed this yesterday at the CAP meeting. We have been using the wrong timeStamp to register the END event for the observing window all this time. The recent change in the name of the variable by Tony has made this clear for us now. We have requested Tony to add a timeStamp for the endofReadout event.

File: etc/ATSCam/primary_hdu.header (hunk 2 above)
Comment (patrickingraham) [reply etc/ATSCam/primary_hdu.header]: Yes. 
"The controller (e.g. O for OCS, C for (A)CCS)

Comment (menanteau) [reply etc/ATSCam/primary_hdu.header]: Thanks for the clarification. Fixed.

**Pull request merged**